.gitignore

```diff
@@ -9,6 +9,8 @@ dist
 *.log
 *.log.*
 *.json
+!playground/deepspeed_config_s2.json
```

Comment:
I don’t think you need this.
You can use “git add xxx.json -f” to add a json file.